cli/deploy-rest.sh

```diff
@@ -63,9 +70,7 @@ curl --location --request PUT "https://management.azure.com/subscriptions/$SUBSC
 --header "Content-Type: application/json" \
 --data-raw "{
   \"properties\": {
-    \"description\": \"Score code\",
-    \"datastoreId\": \"/subscriptions/$SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP/providers/Microsoft.MachineLearningServices/workspaces/$WORKSPACE/datastores/$AZUREML_DEFAULT_DATASTORE\",
-    \"path\": \"score\"
+    \"codeUri\": \"https://maxtest5474417240.blob.core.windows.net/azureml-blobstore-089fff7c-a4c4-42b4-98ef-56a0db069cde/score\"
```

Comment:
can't hardcode paths. Remember that this will get rendered in the public doc: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-deploy-with-rest

Reply:
will remove